meetings/2022-09-08.md

```diff
@@ -44,51 +46,72 @@
 ### Simplify flow diagrams
 URL: https://github.com/solid/notifications/pull/108
 
+* AC: Authorization channel is out of scope, so it doesn't matter in this spec. I took the common parts and made that into one diagram. COuld improve with some relocation of 'Subscriber' in the diagram. The second diagram is to establish the connection for "Connected" client. The last one "Offline" clients and have two versions. Basically have a subscriber, independently of that have target/source happening.
+* RG: +1 on notification source/target. I would leave the connected as is. maybe some gap.
+* SC: +1 on Connected as is. Perhaps update to express resource-specific or storage-wide because right now it says "storage description resource".
+* SC: s/storage-description-resource/notification-channel-resource
```

Comment:
There is no preceding `storage-description-resource`, though there is a preceding `"storage description resource"`, so maybe this?

```suggestion
* SC: +1 on Connected as is. Perhaps update to express resource-specific or storage-wide because right now it says "notification channel resource".
```

Reply:
The replacement suggestion was just a shorthand reference to the diagram (that AC was showing) and not the content of the minutes.

```suggestion
* SC: +1 on Connected as is. Perhaps update to express resource-specific or storage-wide because right now it says "storage description resource".
* SC: Suggest chanting "storage description resource" in the diagram to "notification channel resource"
```